### PR TITLE
Add OTBR on UC tutorial

### PR DESCRIPTION
**Pull request by farshidtz**
A first version with some potential TODOs commented in the text.

### CODE REVIEW

**Comment by MonicaisHer** [comment]
Could you please provide test instructions on how to view the rendered content, either locally or remotely?

**Comment by farshidtz** [comment]
> Could you please provide test instructions on how to view the rendered content, either locally or remotely?

Build instructions are available in the repo itself. There is a make file with all the command.

```
make run
```

File: docs/tutorials/Creating an OS Image with OpenThread Border Router.rst [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+security and bug fixes.
+
+The Ubuntu documentation has an in-depth explanation for Ubuntu Core, covering 
+various concepts. It is recommended to read that documentation, 
+`here <https://ubuntu.com/core/docs/uc20/inside>`_.
+
+Requirements:
+
+- An amd64 Ubuntu development environment
+- A amd64 machine as target for installing the new OS
```
Comment (MonicaisHer) [inline docs/tutorials/Creating an OS Image with OpenThread Border Router.rst:19]:
```suggestion
- An amd64 machine as target for installing the new OS
```

File: docs/tutorials/snapcraft.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+name: otbr-gadget
+type: gadget
+base: core22
+version: test
+summary: OpenThread Border Router Gadget
+description: Custom gadget to configure thr OpenThread Border Router snap
```
Comment (MonicaisHer) [inline docs/tutorials/snapcraft.yaml:6]:
```suggestion
description: Custom gadget to configure the OpenThread Border Router snap
```

File: docs/tutorials/Creating an OS Image with OpenThread Border Router.rst [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+
+To find you developer ID, use the Snapcraft CLI:
+
+.. :code:: bash
+    
+    $ snapcraft whoami
+    ...
+    developer-id: <developer-id>
+
+
```
Comment (MonicaisHer) [inline docs/tutorials/Creating an OS Image with OpenThread Border Router.rst:116]: This content is missing in rendered page

File: docs/tutorials/Creating an OS Image with OpenThread Border Router.rst [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+This will produce a signed model named :code:`model.signed.yaml`.
+
+.. note::
+    You need to repeat the signing every time you change the input model, because
+    the signature is calculated based on the model.
+
+Build the Ubuntu Core image
+---------------------------
+
+We use :code:`ubuntu-image`` and set the path to:
```
Comment (MonicaisHer) [inline docs/tutorials/Creating an OS Image with OpenThread Border Router.rst:167]:
```suggestion
We use :code:`ubuntu-image` and set the path to:
```

File: docs/tutorials/Creating an OS Image with OpenThread Border Router.rst [hunk 5]
```diff
@@ -0,0 +1,230 @@
...
+-----------------------------
+
+The installation instructions are device specific.
+You may refer to Ubuntu Core section in
+`this page <https://ubuntu.com/download/iot>`_. For example:
+
+- `Intel NUC <https://ubuntu.com/download/intel-nuc>`_ - applicable to most computers with a secondary storage
+.. - [Raspberry Pi](https://ubuntu.com/download/raspberry-pi)
+
+A precondition to continue with some of the instructions is to compress
```
Comment (MonicaisHer) [inline docs/tutorials/Creating an OS Image with OpenThread Border Router.rst:193]:
```suggestion
A precondition to continue with some of the instructions is to compress 
```

File: docs/tutorials/Creating an OS Image with OpenThread Border Router.rst [hunk 6]
```diff
@@ -0,0 +1,230 @@
...
+resources, with very little gain.
+
+Now, follow the device specific instructions.
+
+✅ Continue to perform the OS initialization steps appearing `by default`.
+
+Once the installation is complete, you will see the interface of the
+:code:`console-conf` program.
+It will walk you through the networking and user account setup.
+You'll need to enter the email address of your Ubuntu account to create a OS
```
Comment (MonicaisHer) [inline docs/tutorials/Creating an OS Image with OpenThread Border Router.rst:214]:
```suggestion
You'll need to enter the email address of your Ubuntu account to create an OS
```

File: docs/tutorials/Creating an OS Image with OpenThread Border Router.rst (hunk 5 above)
Comment (farshidtz) [reply docs/tutorials/Creating an OS Image with OpenThread Border Router.rst:193]: This isn't needed, as the line-break results in a space. I've double-checked this with the rendered version.

File: docs/tutorials/model.yaml [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+# Setting grade to dangerous to allow use of an unsigned gadget snap
+grade: dangerous
+
+# Since this is a custom model assertion, set the following to your developer ID
+authority-id: <developer-id>
+brand-id: <developer-id>
+
+# Timestamp should be within your signature's validity period, in ISO8601 format
+timestamp: '<timestamp>'
```
[the remark this answers is not in the document]
Comment (farshidtz) [reply docs/tutorials/model.yaml]: I see. The timezone is mandatory in RFC3339. Changed.

**Pull request merged**